frame/staking/src/mock.rs

```diff
@@ -272,6 +273,7 @@ impl crate::pallet::pallet::Config for Test {
 	type GenesisElectionProvider = Self::ElectionProvider;
 	// NOTE: consider a macro and use `UseNominatorsAndValidatorsMap<Self>` as well.
 	type VoterList = BagsList;
+	type Pages = Pages;
```

Comment:
Should we not use instead:

```rust
    type Pages = ConstU32<1>;
```

And remove the previous declaration of `Pages`

Reply:
In one of the tests, I am overwriting this value, so it should be a `static` value.